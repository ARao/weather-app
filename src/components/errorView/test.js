import React from 'react';
import ErrorView from './index'
import { shallow } from 'enzyme'


describe('ErrorView', () => {

  it('++++ render Error View', () => {
    let wrapper = shallow(<ErrorView status="404" statusText="NotFound" />);
    expect(wrapper.length).toEqual(1);
  });

  it('++++ snapshot match', () => {
    let wrapper = shallow(<ErrorView status="401" statusText="UnAuthorised" />);
    expect(wrapper).toMatchSnapshot();
  });
});
