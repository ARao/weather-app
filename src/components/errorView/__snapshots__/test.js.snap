// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ErrorView ++++ snapshot match 1`] = `
<Fragment>
  <div
    className="center"
  >
    <div
      className="pr-3"
    >
      <h1
        className="display-1"
      >
        401
      </h1>
    </div>
    <div>
      <h4
        className="display-4"
      >
        <sub>
          UnAuthorised
        </sub>
      </h4>
    </div>
  </div>
</Fragment>
`;
